lib/model/dto/pokemon_dto.dart

```diff
@@ -9,7 +9,10 @@ part 'pokemon_dto.g.dart';
 class PokemonDto with _$PokemonDto {
   factory PokemonDto({
     required Pokemon pokemon,
-    @Default(0) int id,
+    required int id,
```

Comment:
kindly add a todo that this will be modified once other properties are added for about, stats and moves 🙂